common.mk

```diff
@@ -515,7 +515,7 @@ endif
 ifeq ($(OS_NAME),Darwin)
 # OS X shared library link flags.
 SOFLAGS    := -dynamiclib
-SOFLAGS    += -Wl,-install_name,$(LIBBLIS_SONAME)
+SOFLAGS    += -Wl,-install_name,$(INSTALL_LIBDIR)/$(LIBBLIS_SONAME)
```

Comment:
INSTALL_LIBDIR has DESTDIR in it, but the install name shouldn't have it. I'm not sure what variable gives you the value without the DESTDIR.

Reply:
I think what you're looking for is `$libdir`, which is defined in `config.mk` (created from `build/config.mk.in`):
```
prefix              := /usr/local
exec_prefix         := ${prefix}
libdir              := ${exec_prefix}/lib
includedir          := ${prefix}/include
sharedir            := ${prefix}/share

# Notice that we support the use of DESTDIR so that advanced users may install
# to a temporary location.
INSTALL_LIBDIR      := $(DESTDIR)$(libdir)
INSTALL_INCDIR      := $(DESTDIR)$(includedir)
INSTALL_SHAREDIR    := $(DESTDIR)$(sharedir)

```

Reply:
Yes, thanks.
```suggestion
SOFLAGS    += -Wl,-install_name,$(libdir)/$(LIBBLIS_SONAME)
```